expr/func_aspercent_old_test.go

```diff
@@ -53,8 +53,8 @@ var a1AsPercentOfa1 = []schema.Point{
 }
 var a1AsPercentOfa1b1 = []schema.Point{
 	{Val: 0, Ts: 10}, // 100 * 0 / (0+10)
-	{Val: 100 * float64(50.6) / float64(50.6-25.8), Ts: 20},
-	{Val: 100 * float64(1234567890) / (float64(1234567890) + math.MaxFloat64/1000), Ts: 30},
+	{Val: float64(50.6) / float64(50.6-25.8) * 100, Ts: 20},
+	{Val: float64(1234567890) / (float64(1234567890) + math.MaxFloat64/1000) * 100, Ts: 30},
```

Comment:
these changes make sense. to first divide, then multiply by 100.
multiplying by 100 first as I had it, could lead to overflows